components/ListPageDisplays/ArticleCard.js

```diff
@@ -117,11 +128,19 @@ function ArticleCard({ article, highlight = '' }) {
                 <span>{c('Info box').t`reports`}</span>
               </div>
             </div>
-            <ExpandableText className={classes.content} lineClamp={3}>
-              {highlight
-                ? highlightSections(highlight, highlightClasses)
-                : text}
-            </ExpandableText>
+            {text ? (
+              <ExpandableText className={classes.content} lineClamp={3}>
+                {highlight
+                  ? highlightSections(highlight, highlightClasses)
+                  : text}
+              </ExpandableText>
+            ) : (
+              <img
+                className={classes.attachmentImage}
+                src={attachmentUrl}
+                alt="image"
+              ></img>
+            )}
```

Comment:
nit: since we may add text for images in the future, suggest we do this instead:
```suggestion
            {attachmentUrl && <img
                className={classes.attachmentImage}
                src={attachmentUrl}
                alt="image"
              />}
            {(text || highlight) && (
              <ExpandableText className={classes.content} lineClamp={3}>
                {highlight
                  ? highlightSections(highlight, highlightClasses)
                  : text}
              </ExpandableText>
            )}
```